src/github.rs

```diff
@@ -126,14 +258,17 @@ async fn send_github_issue_query(
     let mut end_cur: Option<String> = None;
     let mut issues: Vec<GitHubIssue> = Vec::new();
     loop {
+        let since_dt = since
```

Comment:
In my point of view, this change(related with `send_github_issue_query`) is not related with `PullRequests`. If it is not directly related, it's best to avoid making this change to prevent unnecessary modifications. Additionally, if this adjustment is necessary, there is a required change concerning the `context`.
- Since the `match` pattern in `fetch_periodically` ensures that a `since` value is always provided, this context is unnecessary and should be replaced with the `?` operator.